openstates_scraped_data_formatter/main.py

```diff
@@ -9,8 +10,13 @@
 # Define state abbreviation and paths
 STATE_ABBR = "il"
 BASE_FOLDER = Path(__file__).parent
-INPUT_FOLDER = BASE_FOLDER / "scraped_state_data" / STATE_ABBR
-DATA_OUTPUT = BASE_FOLDER / "data_output" / STATE_ABBR
+DATA_FOLDER = BASE_FOLDER / "data"
+
+INPUT_FOLDER = os.getenv(
+    "FORMATTER_INPUT_FOLDER", DATA_FOLDER / "scraped_state_data" / STATE_ABBR
+)
+
+DATA_OUTPUT = DATA_FOLDER / "data_output" / STATE_ABBR
```

Comment:
Added another commit, this makes the script a little more configurable with minimal changes, it's used in the github action to the same dir used by the openstates scraper as output.